Sync AirGradient entities with the codebase (#37936)
home-assistant/home-assistant.github.io: source/_integrations/airgradient.markdown

- PM10 density
- Raw nitrogen
- Raw total volatile organic compounds
- Raw PM2.5
- Signal strength
- Temperature
- Total volatile organic compounds index

> A number of configuration entities are available as sensors to automate with if you set up the device via AirGradient instead.

Reviewer: what do you mean by "if you set up the device via AirGradient instead"?

Author: You can set up the device both via the cloud and locally. So if you set airgradient up via the cloud, you can still see the settings of the device, but not as an entity that can be updated using HA

> A number of configuration entities are available as sensors to automate with if you set up the device via AirGradient instead.

Reviewer: ```suggestion
A number of configuration entities are available as sensors to automate with if you set up the device via AirGradient Cloud instead of setting it up locally.
```
is that what you mean?

> A number of configuration entities are available as sensors to automate with if you set up the device via AirGradient Cloud instead of setting it up locally.

Author: ```suggestion
A number of configuration entities are available as sensors to automate with if you control the device via the AirGradient cloud instead of set it to control locally.
```

> A number of configuration entities are available as sensors to automate with if you control the device via the AirGradient cloud instead of set it to control locally.

Author: ```suggestion
A number of configuration entities are available as sensors to automate with if you control the device via the AirGradient dashboard instead of set it to control locally.
```
